LICENSE

```diff
@@ -1,9 +1,25 @@
-Copyright 2017 Open Observatory of Network Interference
+## BSD3 for code:
```

Comment:
I would suggest not using markdown formatting as this is not a markdown file.

Also the license string is `BSD-3-Clause`.

I would say:

"Code is licensed as BSD-3-Clause:"